llvm/lib/ProfileData/InstrProf.cpp

```diff
@@ -497,7 +496,7 @@ Error InstrProfSymtab::create(Module &M, bool InLTO) {
 
 Error InstrProfSymtab::addVTableWithName(GlobalVariable &VTable,
                                          StringRef VTablePGOName) {
-  auto mapName = [&](StringRef Name) -> Error {
+  auto NameToGUIDMap = [&](StringRef Name) -> Error {
```

Comment:
For lambda functions, does code style prefer to make the first character upper case?

Reply:
it's a variable name, so yes (moreover, clang-format marked them in vscode accordingly)